src/control_interface.cpp

```diff
@@ -168,6 +172,33 @@ types::DataPoint<int32_t> getJointPos(robot::types::jointid_t joint) {
 	}
 }
 
+bool hasData(can::deviceid_t id) {
+	auto start = std::chrono::steady_clock::now();
+	const int timeout_ms = 100;
+
+	while (std::chrono::duration_cast<std::chrono::milliseconds>(
+			   std::chrono::steady_clock::now() - start)
+			   .count() < timeout_ms) {
+		if (can::getDeviceTelemetry(id, can::telemtype_t::voltage)) {
+			return true;
+		}
+		std::this_thread::sleep_for(std::chrono::milliseconds(10));
+	}
+	return false;
+}
+
+void verifyAllMotorsConnected() {
+	for (auto motorMap : robot::motorSerialIDMap) {
+		can::pullDeviceTelemetry(std::make_pair(can::devicegroup_t::motor, motorMap.second),
+								 can::telemtype_t::voltage);
+		if (!hasData(std::make_pair(can::devicegroup_t::motor, motorMap.second))) {
```

Comment:
Second time doing `std::make_pair(can::devicegroup_t::motor, motorMap.second)` is here